connexion/exceptions.py

```diff
@@ -60,3 +75,19 @@ def __init__(self, message, reason="Response body does not conform to specificat
 class NonConformingResponseHeaders(NonConformingResponse):
     def __init__(self, message, reason="Response headers do not conform to specification"):
         super(NonConformingResponseHeaders, self).__init__(reason=reason, message=message)
+
+
+class OAuthProblem(ProblemException, Unauthorized):
+    def __init__(self, title='Unauthorized', **kwargs):
```

Comment:
Why do we duplicate here the title? It could come from [`self.name`](https://github.com/pallets/werkzeug/blob/master/werkzeug/exceptions.py#L107).